apps/api_web/lib/api_web/sentry_event_filter.ex

```diff
@@ -0,0 +1,6 @@
+defmodule ApiWeb.SentryEventFilter do
+  @behaviour Sentry.EventFilter
+
+  def exclude_exception?(%Phoenix.Router.NoRouteError{}, :plug), do: true
```

Comment:
question: is this working? I still see a 404 logged after this deployed: https://mbtace.sentry.io/issues/?environment=dev-green&project=4504753674584064&referrer=sidebar&statsPeriod=1h

suggestion: You might also want to ignore 500s from the `/_health` endpoint.

Reply:
Looking into this, it was working locally 🤔

Reply:
(or at least, appeared to be)

Reply:
404: This was occurring in prod but not in dev (mode) because of `handle_sasl_reports: true` on the logger config. I added another rule to handle this case, and it seems to have stopped in `dev-green`. 

503: These errors were actually propagating from the `logster` library. Additionally, because they weren't exceptions Sentry can't filter them. I just disabled the error logging for now, because this will still be visible in Splunk and there's no clean way to filter it. If we ever want the error logging in the future, we can just mute this particular one in the UI.